api/src/types/views/member-info.view.interface.ts

```diff
@@ -4,13 +4,17 @@
  */
 export interface MemberInfoView {
   /** Member's email address */
-  email: string;
+  email?: string;
```

Comment:
I like this, we needed a concept like this in our types.

Reply:
probably more for a discussion.